HW03/Uladzislau_Kanapliou/Uladzislau_Kanapliou_hw_03_t_04_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_relative './Uladzislau_Kanapliou_hw_03_t_04'
+
+describe '.task4' do
+  it 'it return the count of letters and digits in that string' do
```

Comment:
No need to add `it` 2 times 
```suggestion
  it 'returns the count of letters and digits in that string' do
```